src/version.rs

```diff
@@ -4,11 +4,13 @@ use std::ffi::CStr;
 
 // This is the source of truth for the Deno version. Ignore the value in Cargo.toml.
 const DENO_VERSION: &str = "0.1.8";
+const TS_VERSION: &str = "3.1.3";
```

Comment:
It should be read from elsewhere - this might get out of sync.

Reply:
I don't think we are starting any v8 instance during `deno -v`, thus not able to access `ts.version` through binding.
Without hard-coding, we might have to do some kind of injection on rust side during build.

Reply:
It will need to be in userland and:

```ts
import { version } from "typescript";
console.log(version);
```

Reply:
Yea, probably the best way to do this is to send the V8 and Deno versions in the "Start" message to V8, and then print the version from there.